sdk/ml/azure-ai-ml/azure/ai/ml/entities/_deployment/data_collector.py

```diff
@@ -2,12 +2,13 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # ---------------------------------------------------------
 
-from typing import Any, Dict, Optional, Mapping
+from typing import Dict, Optional
 
```

Comment:
Please create a backlog to include a test to verify the schema generation on the PR's where the schema is modified.

Reply:
I created a work item and assigned it to Taji to keep track of this work.